src/components/ImportExport.js

```diff
@@ -115,22 +150,36 @@ class ImportExport extends React.Component {
 
   exportFile = () => {
     const { trails, hydrants } = this.props;
-    const { selectedExport } = this.state;
+    const { selectedExport, exportType } = this.state;
+
 
     const trailFeatures = _.flatMap(_.values(trails.toJS()), item => item.features);
-    const hydrantFeatures = _.flatMap(_.values(hydrants.toJS()),item => item.features);
+    const hydrantFeatures = _.flatMap(_.values(hydrants.toJS()), item => item.feature);
+    const ext = exportType === 'GJ' ? 'json' : 'kml';
+
+    hydrantFeatures.forEach((f,i) => {
+      const desc = `${f.get('trailName')},${i},${f.get('name')}`
+      f.set('description', desc)
+    })
 
+    trailFeatures.forEach((feature) => {
+      //Features in a feature creates GeoJson Circular stringify error
+      feature.unset('features')
+      feature.set('description', feature.get('name'))
```

Comment:
Lance's `description` in the KML's currently is the name and a number (probably number of hydrants), separated by a comma. 

Also, the name is inconsistently formatted (snake_casing, usually capitalized, but I don't think always?). As a temporary thing to keep with his current system (and since adding new trails to existing installations in the next few weeks is unlikely), let's do something similar with what we do for hydrants; set an `originalName` on the trail on import that we know won't change, and use that in the exports. Also, we can use that name when exporting the hydrants to ensure it's the same (eg. for a given hydrant, `legacyTrailName = trails.getIn([hydrant.get('trail'), 'features', 0]).get('originalName')`

Reply:
Taken Care of in PR #62